domain/metadata/token_metadata.py

```diff
@@ -0,0 +1,88 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+from domain.metadata.metadata import Metadata, MetadataType
+
+
+class TokenNFTMediaType(str, Enum):
+    VIDEO = 'VIDEO'
+    IMAGE = 'IMAGE'
+    AUDIO = 'AUDIO'
+
+
+@dataclass
+class TokenNFTMedia:
+    """Data of token NFT
+
+    :param type: NFT Type. image or video.
+    :type type: :py:class:`domain.metadata.token_metdata.TokenNFTMediaType`
+
+    :param file: Media file of the NFT
+    :type file: str
+
+    :param loop: If media will play in loop or not
```

Comment:
Works only for video medias? Or should we have this for audio as well?

We should make explicit here and in the token meta example in the RFC

Reply:
For audio too. Ok added more explanation